eras/conway/impl/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ## 1.7.0.0
 
+* Add `insertGovActionsState`
```

Comment:
Just realized something.

This entry now needs to go to the next version above and it needs to be switched to `1.8.0.0` (both in cabal file and in changelog)

Change in this is breaking, since `GovActionState` and `RatifyState` types has changed, which should also go into the change log:

```markdown
* `GovActionState` now has new field `gasId`
* Type for `rsRemoved` in `RatifyState` was changed.
```

No need to mention `rsFuture`, since it will be removed in a subsequent PR